bake/bake_test.go

```diff
@@ -610,7 +610,7 @@ func TestHCLContextCwdPrefix(t *testing.T) {
 	require.Equal(t, 1, len(m))
 	require.Contains(t, m, "app")
 	assert.Equal(t, "test", *m["app"].Dockerfile)
-	assert.Equal(t, "foo", *m["app"].Context)
+	assert.Equal(t, "cwd://foo", *m["app"].ContextPath())
```

Comment:
> `cwd://foo`

Was this a bug or is it a behavior change? I think a `path.Clean(bi.ContextPath)` is missing in this else cond: https://github.com/docker/buildx/pull/2906/files#diff-fbe2fa23203f2ff05f06b6c0731ea90529f90906807483db76a79f6018781465R1422